### ISSUE TITLE
Splash page - status bar on top of the app not readable (build 58)

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#795]
Just on splash page, the top area of the screen (status bar) that has notifications and phone information is not much visible. 
Other pages are fine.

Android 8.1.0 on Samsung Galaxy
iOS 13.5.1 on iPhone 7

![image](https://user-images.githubusercontent.com/14280062/87955933-63f55780-ca7c-11ea-97ae-92a532d58b9c.png)

![image](https://user-images.githubusercontent.com/14280062/87956316-e54cea00-ca7c-11ea-8987-05f9a48a8b3a.png)
